.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
+/.build
+/Packages
+xcuserdata/
+DerivedData/
+.swiftpm/configuration/registries.json
+.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata
```

Comment:
why no the entire .swiftpm folder?

Reply:
I think certain files in `.swiftpm` act like a `Package.lock` file for managing dependencies, so I wanted to keep those in.

Reply:
That makes sense.
However, I am not seeing them in the repo.
If you need them, they need to be added I think.

Reply:
Ah that's true, not really much in here. I think this was the boilerplate `.gitignore` set up for me by XCode though, so I'm a little hesitant to change it.